src/components/Emoji/index.ts

```diff
@@ -0,0 +1,12 @@
+/* Internal dependencies */
+import Emoji from './Emoji'
+import EmojiProps, { EmojiSize } from './Emoji.types'
```

Comment:
```suggestion
import { EmojiSize } from './Emoji.types'
import type EmojiProps from './Emoji.types'
```
`import type`구문을 사용하게 되면 컴파일 시에 타입 선언을 더 잘 지워줍니다~